lib/facter/conjur.rb

```diff
@@ -11,8 +18,23 @@ def config
       end
     end
 
+    def config_registry
+      keyname = 'Software\\CyberArk\\Conjur'
```

Comment:
Can this be a constant instead? I know the environment is weird inside the `setcode` block, so I'm not actually sure :)
BTW, you don't need to double the backslashes inside single quotes afaik.